bindings/python/pinocchio/utils.py

```diff
@@ -34,18 +35,24 @@ def eye(n):
     if pin.getNumpyType()==np.matrix:
         return np.matrix(res)
     else:
+        message = "This function is deprecated for numpy.array. Plase use numpy.eye"
+        _warnings.warn(message, category=DeprecatedWarning, stacklevel=2)
         return res
 
 def zero(n):
     if pin.getNumpyType()==np.matrix:
         return np.matrix(np.zeros([n, 1] if isinstance(n, int) else n))
     else:
+        message = "This function is deprecated for numpy.array. Please use numpy.zeros"
+        _warnings.warn(message, category=DeprecatedWarning, stacklevel=2)
         return np.zeros(n)
 
 def rand(n):
     if pin.getNumpyType()==np.matrix:
         return np.matrix(np.random.rand(n, 1) if isinstance(n, int) else np.random.rand(n[0], n[1]))
     else:
+        "This function is deprecated for numpy.array. Please use numpy.random.rand"
```

Comment:
A small remark here, I don't think that we should warn here. I mean, we can provide shortcut.

Reply:
I think that in the case of `array` these functions are entirely redundant and in the long run they should be eliminated as the support for matrix is dropped.
However, this is just a personal feeling. If you disagree, it is a single isolated commit, you can certainly revert it if you want

Reply:
I will revert it. I think it will provide too much pollution.